src/components/TodoList.js

```diff
@@ -3,17 +3,15 @@ import TodoForm from './TodoForm';
 import Todo from './Todo';
 
 function TodoList() {
-    const [todos, setTodos] = useState([]);
+    const [todos, setTodos] = useState({});
 
     const LOCAL_STORAGE_KEY = "react-do-list-todos";
 
     useEffect(() => {
-        const storageTodos = JSON.parse(localStorage.getItem(LOCAL_STORAGE_KEY));
```

Comment:
JSON.parse is must, as in localstorage all the values are stored as plain strings